src/platform/plugins/shared/dashboard/public/dashboard_app/url/search_sessions_integration.ts

```diff
@@ -79,18 +79,12 @@ function getLocatorParams({
 }): DashboardLocatorParams {
   const savedObjectId = dashboardApi.savedObjectId$.value;
 
-  const panels = dashboardInternalApi.serializeLayout() as Pick<
+  const { panels, controlGroupInput, references } = dashboardInternalApi.serializeLayout() as Pick<
     DashboardLocatorParams,
-    'panels' | 'references'
+    'panels' | 'controlGroupInput' | 'references'
   >;
-
-  // TODO Update serializing controls https://github.com/elastic/kibana/issues/242768
-  // const { controlGroupInput, controlGroupReferences } = dashboardInternalApi.serializeControls();
-
-  const combinedReferences = [
-    ...(panels?.references ?? []),
-    // ...(controlGroupReferences ?? []),
-  ] as unknown as DashboardState['references'] & SerializableRecord;
+  const combinedReferences = [...(references ?? [])] as unknown as DashboardState['references'] &
```

Comment:
Do we still need this line? Can we not pass it through directly now?

Reply:
Good catch - done in 039095dc111d535eab7a062d11a4e3425bf65120